### DISCUSSION TITLE
[FAQ] When is it appropriate to use Velero instead of etcd's built in backup/restore?

### DISCUSSION THREAD

**Discussion by dsu-igeek** [vmware-tanzu/velero#3367] [Community support Q&A] [answered]
When is it appropriate to use Velero instead of etcd's built in backup/restore?

**Comment by dsu-igeek** [comment, marked as answer]
Etcd's backup/restore tooling is good for recovering from data loss in a single etcd cluster. For example, it is a good idea to take a backup of etcd prior to upgrading etcd itself. For more sophisticated management of your Kubernetes cluster backups and restores, we feel that Velero is generally a better approach. It gives you the ability to throw away an unstable cluster and restore your Kubernetes resources and data into a new cluster, which you can't do easily just by backing up and restoring etcd.

Examples of cases where Velero is useful:

- you don't have access to etcd (e.g. you're running on GKE)
- backing up both Kubernetes resources and persistent volume state
- cluster migrations
- backing up a subset of your Kubernetes resources
- backing up Kubernetes resources that are stored across multiple etcd clusters (for example if you run a custom apiserver)

**Comment by jayunit100** [comment]
Im not 100% sure, but Im assuming velero is also smart enough to only back up the "spec" of things, and not add in old "status" info. 

A raw etcd dump would bring with it old `status` data from the previous cluster that isnt relevant post-migration.
